src/lib/gssapi/mechglue/mglueP.h

```diff
@@ -82,16 +82,13 @@ int gssint_mechglue_initialize_library(void);
 OM_uint32 gssint_get_mech_type_oid(gss_OID OID, gss_buffer_t token);
 
 /*
- * This is the definition of the mechs_array struct, which is used to
- * define the mechs array table. This table is used to indirectly
- * access mechanism specific versions of the gssapi routines through
- * the routines in the glue module (gssd_mech_glue.c)
+ * This table is used to indirectly access mechanism specific versions of the
+ * gssapi routines through the routines in the glue module (g_glue.c)
  *
- * This contants all of the functions defined in gssapi.h except for
- * gss_release_buffer() and gss_release_oid_set(), which I am
- * assuming, for now, to be equal across mechanisms.
+ * This contains all of the functions defined in gssapi.h except for
+ * gss_release_buffer() and gss_release_oid_set(), which are assumed to be
+ * identical across mechanisms.
```

Comment:
I would combine these paragraphs, starting the second sentence with "It contains...".